R/utils_render_grid.R

```diff
@@ -1408,7 +1408,7 @@ parse_css <- function(data) {
   classes <- Map(`:`, start + 1, end - 1)
   names(classes) <- names
   classes <- lapply(classes, function(x) unlist(split[x], FALSE))
-  classes <- classes[grepl("^gt_", names(classes))]
+  classes <- grep("^gt_", names(classes), value = TRUE)
```

Comment:
```suggestion
  classes <- classes[grepl("^gt_", names(classes))]
```

We want classes, not `names(classes)`. This was the main bug

Reply:
```suggestion
  classes <- classes[startsWith(names(classes), "gt_")]
```
This may work too, but I haven't tested it.

Reply:
Argh, I suspected that line from the beginning but I re-read it 4+ times without spotting the mistake!